Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:5.6-cli
+
+WORKDIR /usr/src/app/
+
+COPY composer-install.sh /usr/src/app/
+
+RUN apt-get update
+RUN apt-get install wget
+RUN ./composer-install.sh
```

Comment:
Should we be using `.wp-env`/`@wordpress/env` instead of custom docker containers as you get the WP environment ready and you'd be able to use it for all tests and checks and even for development.

Reply:
@youknowriad I assume you are referring to this? https://make.wordpress.org/core/2020/03/03/wp-env-simple-local-environments-for-wordpress/

@raaaahman it might be worthwhile investigating this as an option. What I do like about it is it allows folks already familiar with the tool to help us develop and test

Reply:
Thank you @youknowriad , this environment looks promising! I made the minimum necessary to use it here: https://github.com/WordPress/wp-notify/pull/22

For now though, we don't need a full WordPress website to be loaded. But we will soon!

Do you have any idea about when PHP 8 will be available?